pyaedt/modeler/cad/Modeler.py

```diff
@@ -5580,3 +5580,305 @@ def wrap_sheet(self, sheet_name, object_name, imprinted=False):
         if imprinted:
             self.cleanup_objects()
         return True
+
+    @pyaedt_function_handler()
+    def heal_objects(
+        self,
+        input_objects_list,
+        auto_heal=True,
+        tolerant_stitch=True,
+        simplify_geometry=True,
+        tighten_gaps=True,
+        heal_to_solid=False,
+        stop_after_first_stitch_error=False,
+        max_stitch_tolerance=0.001,
+        explode_and_stitch=True,
+        geometry_simplification_tolerance=1,
+        maximum_generated_radius=1,
+        simplify_type=0,
+        tighten_gaps_width=0.00001,
+        remove_silver_faces=True,
+        remove_small_edges=True,
+        remove_small_faces=True,
+        silver_face_tolerance=1,
+        small_edge_tolerance=1,
+        small_face_area_tolerance=1,
+        bounding_box_scale_factor=0,
+        remove_holes=True,
+        remove_chamfers=True,
+        remove_blends=True,
+        hole_radius_tolerance=1,
+        chamfer_width_tolerance=1,
+        blend_radius_tolerance=1,
+        allowable_surface_area_change=5,
+        allowable_volume_change=5,
+    ):
+        """Repair invalid geometry entities for the selected objects within the specified tolerance settings.
+
+        Parameters
+        ----------
+        input_objects_list : str
+            List of object names to analyze.
+        auto_heal : bool, optional
+            Auto heal option. Default value is ``True``.
```

Comment:
Any chance you'd be willing to do a search and replace to format the sentence with the default like this: "The default is ...." We'd like to keep the formatting consistent within and across all libraries. Thanks.

Reply:
done, thanks!